app/views/energy_tariffs/energy_tariff_differential_prices/index.html.erb

```diff
@@ -32,6 +32,11 @@
           <%= t('schools.user_tariff_differential_prices.index.incomplete_message') %>.
         <% end %>
       <% end %>
+      <% if @energy_tariff.energy_tariff_prices.invalid_prices? %>
```

Comment:
Can I suggest a display tweak here, currently if there's missing prices you'll get a positive notice about full 24 hour coverage, but an error about missing prices.

I think it might be better to only show the positive notice (that prices and coverage are correct) if all the negatives have been addressed.